pkg/store/diskstore/diskstore.go

```diff
@@ -34,8 +35,10 @@ func lastFile(files []os.FileInfo) os.FileInfo {
 	return nil
 }
 
-func (s *diskStore) ReadMetrics(ctx context.Context, minTimestampMs int64, fn func(partitionKey string, families []*clientmodel.MetricFamily) error) error {
-	return filepath.Walk(s.path, func(path string, info os.FileInfo, err error) error {
+func (s *diskStore) ReadMetrics(ctx context.Context, minTimestampMs int64) ([]*store.PartitionedMetrics, error) {
```

Comment:
Telemeter does not use this code the diskstore at all and has no entry-point to it, so I would suggest we simply remove this package entirely.

Reply:
deleted code is the best code :+1: